### PR TITLE
Update appui-test-app README.md

### PR DESCRIPTION
**Pull request by KaroAug**
## Changes

Updated links to open the right file in repository.

Since there is no "buildapp" script, changed to "build". 

## Testing

N/A

### CODE REVIEW

File: test-apps/appui-test-app/connected/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The application contained within this directory provides a test environment for
 
 The application may be run as an Electron app, Mobile app or within a browser. The following steps outline the procedure for successfully building the application as part of a larger monorepo, and then starting the application via npm scripts.
 
-1. To get started, follow the instructions to setup the entire repository, located [here](<../../README.md#Build\ Instructions>).
+1. To get started, follow the instructions to setup the entire repository, located [here](<../../../README.md#Build\ Instructions>).
```
Comment (GerardasB) [inline test-apps/appui-test-app/connected/README.md:11]: The anchor link doesn't seem to be working.
```suggestion
1. To get started, follow the instructions to setup the entire repository, located [here](../../../README.md#build-instructions).
```

File: test-apps/appui-test-app/standalone/README.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ The application contained within this directory provides a test environment for
 
 The application may be run as an Electron app, Mobile app or within a browser. The following steps outline the procedure for successfully building the application as part of a larger monorepo, and then starting the application via npm scripts.
 
-1. To get started, follow the instructions to setup the entire repository, located [here](<../../README.md#Build\ Instructions>).
+1. To get started, follow the instructions to setup the entire repository, located [here](<../../../README.md#Build\ Instructions>).
 2. Optionally, set other environment variables to configure the application prior to startup. The full list of supported variable are [below](#environment-variables).
-3. Build the application code using `buildapp` script (`build` is reserved for rush overall builds, we use a different script to build this application backend to reduce normal builds):
+3. Build the application code using `build` script:
```
Comment (GerardasB) [inline test-apps/appui-test-app/standalone/README.md:13]: I think we can get rid of this step due to `1.`

**Review by GerardasB — CHANGES_REQUESTED** [review]

File: test-apps/appui-test-app/standalone/README.md (hunk 2 above)
Comment (KaroAug) [reply test-apps/appui-test-app/standalone/README.md:13]: Should we get rid of it in "test-apps/appui-test-app/connected/README.md" too?

Comment (GerardasB) [reply test-apps/appui-test-app/standalone/README.md:13]: Yes, we should.

**Review by GerardasB — APPROVED** [review]

**Pull request merged**